### PR TITLE
[SYNC-227][FE] Refactor and style footer

### PR DESCRIPTION
**Pull request by andyrochi**
## Description: 
- Implement new footer
## Changes:
- Remove styles not needed for simplicity
- New UI
## Test Scope:
- http://localhost:8080/#/
## Screenshots (optional)
image
image

image

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -57,14 +27,16 @@ export default {
...
-.footer-button {
-  font-size: 20px;
-  font-family: 'Noto Sans CJK TC';
-  color:#B3B3B3;
-  letter-spacing: 6px;
+  height: 400px;
+  padding-left: 64px;
+  padding-right: 64px;
+  padding-top: 80px;
+  padding-bottom: 72px;
```
Comment (david20571015) [inline src/components/Footer.vue:34]: Is it better to use [shorthand properties](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)?

File: src/components/Footer.vue [hunk 2]
```diff
@@ -77,29 +49,40 @@ export default {
...
 }
 
-@media only screen and (min-width: 990px) {
+@media only screen and (max-width: 679px) {
   .footer {
-    height: 421px;
+    padding-top: 48px;
+    padding-bottom: 48px;
+    padding-left: 20px;
+    padding-right: 20px;
```
Comment (david20571015) [inline src/components/Footer.vue:80]: Same.

File: src/components/Footer.vue (hunk 1 above)
Comment (yccodr) [reply src/components/Footer.vue:34]: It might be more concise in the shorthand version, IMO. But it's totally fine if you leave it like this.

**Review by yccodr — APPROVED** [review]

File: src/components/Footer.vue (hunk 1 above)
Comment (andyrochi) [reply src/components/Footer.vue:34]: I did think about using shorthand properties, but I find it easier to identify the properties lool.
Would fix it.

**Pull request merged**